internal/envoy/envoy.go

```diff
@@ -319,17 +397,22 @@ func (srv *Server) parseLog(line string) (name string, logLevel string, msg stri
 
 func (srv *Server) handleLogs(rc io.ReadCloser) {
 	defer rc.Close()
+
+	bo := backoff.NewExponentialBackOff()
+
 	s := bufio.NewReader(rc)
 	for {
 		ln, err := s.ReadString('\n')
 		if err != nil {
-			if err == io.EOF {
+			if errors.Is(err, io.EOF) || errors.Is(err, os.ErrClosed) {
```

Comment:
Saw this come up when the process was closed and spam stdout. We should now treat `ErrClosed` like `EOF`, but I also added exponential backoff to avoid the infinite loop if another error happens.

Reply:
👍